src/views/Profile.tsx

```diff
@@ -0,0 +1,21 @@
+import { useLoaderData } from 'react-router-dom';
+
+type Profile = {
+  id?: string;
```

Comment:
Is this a possibility? Perhaps we should prevent the route from being hit at all if there is no parameter supplied

Reply:
The route returns the Next error page if a user attempts to access /profile/ without a param. I think I changed it to be an optional parameter before I added the default id value to the loader as it was giving me TS errors, but then I didn't remove the optional mark after.